sdk/anomalydetector/ai-anomaly-detector/karma.conf.js

```diff
@@ -26,7 +26,7 @@ module.exports = function(config) {
       "karma-ie-launcher",
       "karma-env-preprocessor",
       "karma-coverage",
-      "karma-remap-istanbul",
+      "karma-sourcemap-loader",
```

Comment:
Thanks, @xirzec, this is amazing!

I've been meaning to try karma-typescript instead, have you tried that too?

Reply:
karma-typescript expects to read in TS files and do the compilation -- but since this is a browser bundle, we really want rollup to make the test js file for us with all the browser versions of modules. Not sure that we can do this with karma-typescript.

Reply:
Yes, it takes TS files. But, since we are not publishing the browser bundle anyway, what's the point of making a bundle?

Is the idea that it would closely represent how the user may do the bundling?

Reply:
The bundle makes sure our browser versions of files work as expected. And you can't really test without bundling in some form since we're not really using es module support in the browser yet.